src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -248,11 +217,12 @@ export class MeetingManager implements AudioVideoObserver {
       this.audioVideo.unbindAudioElement();
 
       try {
-        await this.audioVideo.chooseVideoInputDevice(null);
-        await this.audioVideo.chooseAudioInputDevice(null);
-        await this.audioVideo.chooseAudioOutputDevice(null);
+        await this.meetingSession?.deviceController.destroy();
+        await this.audioVideo.chooseAudioOutput(null);
```

Comment:
Can we reverse the order here? Weird to call destroy then try to call `chooseAudioOutput`.

Reply:
Updated to:
```js
await this.meetingSession?.deviceController.chooseAudioOutput(null);
await this.meetingSession?.deviceController.destroy();
```